packages/openactive-integration-tests/test/helpers/flow-stages/order-feed-update.js

```diff
@@ -203,6 +230,12 @@ const OrderFeedUpdateFlowStageUtils = {
       uuid: orderFeedUpdateParams.uuid,
       logger: orderFeedUpdateParams.logger,
       orderFeedType,
+      failEarlyIf: () => {
+        if (failEarlyIf()) { return true; }
+        // Don't bother collecting if the wrapped stage fails.
+        if ((wrappedStage.getOutput()?.httpResponse?.response?.statusCode ?? 200) >= 400) { return true; }
```

Comment:
Also added this bit to automatically fail if the wrapped stage fails. This needs to be done automatically rather than with the `failEarlyIf` arg, because the wrapped stage isn't accessible to the caller of `OrderFeedUpdateFlowStageUtils.wrap(..)` because `OrderFeedUpdateFlowStageUtils.wrap(..)` creates the wrapped stage.

This also means that this doesn't just fix timeouts after P fails. It should fix all fail -> listen for Order Feed update timeouts